.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,30 +1,37 @@
-**By submitting this pull request, I confirm the following:** 
-*please fill any appropriate checkboxes, e.g: [X]*
+## Thank you for your contribution to the Pi-hole Community! 
+
+Please read the comments below to help us consider your Pull Request.
+
+We are all volunteers and completing the process outlined will help us review your commits quicker.
 
-- [ ] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md), as well as this entire template.
-- [ ] I have made only one major change in my proposed changes.
-- [ ] I have commented my proposed changes within the code.
-- [ ] I have tested my proposed changes, and have included unit tests where possible.
-- [ ] I am willing to help maintain this change if there are issues with it later.
-- [ ] I give this submission freely and claim no ownership.
-- [ ] It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
-- [ ] I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))
+**Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits and base your code and PRs against the repositories developmental branch.**
 
-Please make sure you [Sign Off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/contributing/).
+We enforce the [DCO](https://docs.pi-hole.net/guides/github/contributing/) for all contributions.
```

Comment:
```suggestion
We enforce the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions. Please [sign all your commits](https://docs.pi-hole.net/guides/github/how-to-signoff/).
```

Reply:
I think that will be confusing to have the DCO and the commit signoffs so close to each other.